### PR TITLE
[Cost-2729]Local error - Task failed: 'NoneType' object has no attribute 'id'

### PR DESCRIPTION
**Pull request by Fozzywozzybear**
[COST-2729](https://issues.redhat.com/browse/COST-2729)

## Description

Changes proposed in this PR:

Stop processing when OCP report period is none, (returns start_date and end_date)
## Testing

1. Set for TRINO processing
2. Execute summary processing for a start_date for a provider_uuid having no report_period data
3. No report period id will be logged and summary processing will not execute

## Notes

...

### CODE REVIEW

File: koku/masu/processor/ocp/ocp_report_parquet_summary_updater.py [hunk 1]
```diff
@@ -113,6 +113,9 @@ def update_summary_tables(self, start_date, end_date):
         with OCPReportDBAccessor(self._schema) as accessor:
             with schema_context(self._schema):
                 report_period = accessor.report_periods_for_provider_uuid(self._provider.uuid, start_date)
+                if not report_period:
+                    LOG.info(f"No report period for {self._provider.uuid} with start date {start_date}")
```
Comment (Red-HAP) [inline koku/masu/processor/ocp/ocp_report_parquet_summary_updater.py:117]: **Question**

Should this be a warning? Should the log message also indicate that no processing has occurred?

Comment (Red-HAP) [reply koku/masu/processor/ocp/ocp_report_parquet_summary_updater.py:117]: Should it be a `LOG.warning` instead of a `LOG.info`?

Comment (Fozzywozzybear) [reply koku/masu/processor/ocp/ocp_report_parquet_summary_updater.py]: Log info has been replaced with warning.

**Review by Red-HAP — APPROVED** [review]

**Pull request merged**